Documentation/building/linux-instructions.md

```diff
@@ -37,7 +40,7 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev`
+`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev curl libcurl4-openssl-dev libssl-dev`
```

Comment:
>  had wget installed, but when running ./build.sh native (corefx) I hit an error that curl was not installed 

I am not sure how that is possible.

In short, if you can go back to the initial state (where you only had wget and no sign of curl), then create a test script `nano ./test.sh` with the following content:

```bash
#!/usr/bin/env bash

hash curl 2>/dev/null || hash wget 2>/dev/null || {
  echo >&2 echo "cURL or wget both are not available. You really need help dude!" ;
  exit 1;
}

which curl > /dev/null 2> /dev/null

if [ $? -ne 0 ]; then
  echo "wget is available"
else
  echo "curl is available"
fi
```

after that: `chmod +x ./test.sh` and run `./test.sh`, it should show you "wget is available". This is the ditto construct used in `./build.sh` script and i verified it on three different Unices: Ubuntu, CentOS and FreeBSD, which is the reason of my doubt.